### ISSUE TITLE
Resizing increases line width of strokes

### ISSUE THREAD

**Issue by itsCryne** [flxzt/rnote#1017]
**Describe the bug**  
Repeatedly resizing brush or shape paths leads to them increasing their thickness

**To Reproduce**  
1. Draw something using the Brush or Shaper
2. Select it
3. Repeatedly resize it

**Expected behavior**  
scale1 == scale2 <=> thickness1 == thickness2

**Console Output**  
```
 2024-02-13T14:28:39.745Z DEBUG rnote > ... env_logger initialized

(rnote:287344): Gtk-CRITICAL **: 15:28:40.110: gtk_widget_class_set_accessible_role: assertion '!gtk_accessible_role_is_abstract (accessible_role)' failed
 2024-02-13T14:28:40.128Z DEBUG rnote_engine::store > state has not changed, no need to record

(rnote:287344): Gtk-CRITICAL **: 15:29:13.510: The layout manager of type GtkBoxLayout 0x5653416400b0 does not create GtkLayoutChild instances
```

**Screenshots**  

https://github.com/flxzt/rnote/assets/51213332/0c87ab41-4dcc-424f-a5c3-51fa784ac476

**Desktop (please complete the following information):**  
 - OS: Manjaro
 - App Version: Tested with Rnote v0.9.4 & 551b1c5ed141d5089d851a11771387f832269e87
 - Installation Source: Arch package & built from source
 - Desktop Environment: GNOME 45.3
 - Display Server: Wayland
 - Input Source: Mouse and Keyboard, On-Screen-Tablet
 
**Additional context**  
\-

**Comment by Doublonmousse** [comment]
The code that changes line width based on scale was probably implemented in #808.
I think the line width is changed in place when resizing occurs, so that it can get to these large sizes, instead of staying as a `width * (x+y)/2` throughout the resize action

**Comment by Doublonmousse** [comment]
Basically, for each step of the resize movement, the scaling factor is applied with `width <- width * (x+y)/2` with `x` and `y` the scaling ratio for both axis with respect to the last one. 

Unless you lock the ratio, changing the ratio creates this cumulative effect.

Could the resize take as `scale` the ratio with respect to the initial size of the selection bound (so we need to save it) and apply the width with respect to the initial line width of the selected strokes (that has to be saved as well for as long as elements are selected) ?

This way, resizing will be more predictable and still keep the instant preview feature (the line stroke corresponds to the final one, unlike xournal)
